### PR TITLE
Scalar Falsi

### PR DESCRIPTION
**Pull request by kanav99**
@ChrisRackauckas this one is without special AD fixes, only new Scalar Falsi that works

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ ForwardDiff = "0.10.3"
 RecursiveArrayTools = "2"
 Reexport = "0.2"
 Setfield = "0.7"
-StaticArrays = "1.0"
+StaticArrays = "1.0 0.12"
```
Comment (ChrisRackauckas) [inline Project.toml:21]:
```suggestion
StaticArrays = "0.12,1.0"
```

that's not well-formed?

**Comment by ChrisRackauckas** [comment]
The tests fail locally.

**Comment by kanav99** [comment]
Strange, they do pass for me. Can you paste the error?

**Comment by ChrisRackauckas** [comment]
I'll trust CI. I have a ton of dev branches.

**Pull request merged**